docs/src/development.md

```diff
@@ -0,0 +1,74 @@
+# Snowflake development
+
+## Installing Snowflake for local development
+
+When developing Snowflake, you must ensure that you are using a local copy of Snowflake, not the latest released version. The easiest way to achieve that is to set the project to the local directory.
+
+If you are starting a new instance of Julia, then you can activate the Snowflake project with
+
+```bash
+julia --project=.
+```
+
+Or, if you are inside a script or interpreter, you can use
+
+```julia
+using Pkg
+Pkg.activate(".")
+```
+
+If the current directory is not the Snowflake project, replace `.` with the Snowflake project path.
+
+
+## Running tests
+
+First open a julia REPL in the current project
+
+```bash
+julia --project=.
+```
+
+and run the tests
+
+``julia
+using Pkg
+Pkg.test()
+```
+
+## Build the documentation
+
+First open a julia REPL using the docs project
+
+```bash
+julia --project=./docs
+```
+
+If it is the first time building the docs you need to instantiate the Julia project and add the Snowflake project as a development dependency.
```

Comment:
This means the version of the Snowflake package loaded is the one at the path specified (pwd()), not the one  registered at JuliaHub.